docs/recipes/jspm-systemjs.md

```diff
@@ -14,11 +11,15 @@ This recipe has only been tested with JSPM v0.17.0-beta.22, but it should work w
 
 ### Babel
 
-Configure your .babelrc to work with AVA if you have not already. NOTE: You can keep additional configuration in your JSPM config files to override these settings during bundling and building.
+Configure your .babelrc to work with AVA if you have not already. NOTE: You can keep additional configuration in your JSPM config files to override these settings during bundling and building. As of babel 7 [@babel/preset-env](https://www.npmjs.com/package/@babel/preset-env) will transform code as babel-preset-esXXXX did previously.
```

Comment:
I don't think we need to explain the virtues of `@babel/preset-env`.

Reply:
@novemberborn Personally, I find the indication useful. It allows to know that the behavior is the same.

Reply:
Yes but IMHO that's part of Babel 7's upgrade documentation, not AVA's.

Reply:
OK. I agree with you